tests/web/Ticket.php

```diff
@@ -36,31 +36,36 @@
 namespace tests\units;
 
 use Glpi\Toolbox\Sanitizer;
-use Symfony\Component\BrowserKit\HttpBrowser;
 
 class Ticket extends \FrontBaseClass
 {
     public function testTicketCreate()
     {
         $this->logIn();
+
         $this->addToCleanup(\Ticket::class, ['name' => ['LIKE', '%thetestuuidtoremove']]);
 
-        //load computer form
-        $crawler = $this->http_client->request('GET', $this->base_uri . 'front/ticket.form.php');
+        //load ticket form
+        $this->http_client->request('GET', $this->base_uri . 'front/ticket.form.php');
+
+        $crawler = $this->http_client->waitFor('form[name=itil_form]');
+
+        // Cannot find a way to make TinyMCE works properly.
+        // Destroy TinyMCE instance to be able to define textarea contents.
+        $tinyid = $crawler->filter('textarea[name=content]')->attr('id');
+        $this->http_client->executeScript("tinymce.get('$tinyid').destroy();");
```

Comment:
As said with @cedric-anne in our private discussion channel; I'm not really fan of disabling TinyMCE; even if I do not see any other solution.
It could lend to impossible te reproduce use cases, or test that will work only if it's not enabled (which does not happen IRL).
Also, panther causes issues with date pickers, dropdowns (for what I've been able to see), if the solution is the same, that's really not a good point.

Globally, I'm not really convinced with the whole solution; I'm not going to spend more time on that "feature".